xml/sle_update_offline.xml

```diff
@@ -0,0 +1,599 @@
+   sure that the following requirements are met:
+  </para>
+  <variablelist>
+   <title>Requirements for Upgrading from a Network Installation Source</title>
+   <varlistentry>
+    <term>Network Installation Source</term>
+    <listitem>
+     <para>
+      A network installation source is set up according to
+      <xref linkend="cha.deployment.instserver"/>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Network Connection and Network Services</term>
+    <listitem>
+     <para>
+      Both the installation server and the target machine have a functioning
+      network connection. The network must provide the following services: a
+      name service, DHCP (optional, but needed for booting via PXE), and
+      OpenSLP (optional).
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Installation Media</term>
+    <listitem>
+     <para>
+      You have a &sle; DVD 1 (or a local ISO image) at hand to boot the target
+      system <emphasis>or</emphasis> a target system that is set up for
+      booting via PXE according to
+      <xref linkend="sec.deployment.prep_boot.pxeprep" xrefstyle="HeadingOnPage"/>.
+      Refer to <xref linkend="cha.deployment.remoteinst"/> for in-depth
+      information on starting the upgrade from a remote server.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   When upgrading from network installation source, you can either boot from
+   the local medium and then select the respective network installation type,
+   or boot via PXE. Select the method of your choice and proceed as described
+   in
+   <xref linkend="pro.update.sle.manual.network.boot-from-dvd" xrefstyle="select:label"/>
+   or
+   <xref linkend="pro.update.sle.manual.network.pxe-boot" xrefstyle="select:label"/>.
+  </para>
+  <procedure xml:id="pro.update.sle.manual.network.boot-from-dvd">
+   <title>Manually Upgrading from &slea;&nbsp;11 SP3 or SP4 to &slea;&nbsp;12 SP2 via Network Installation Source&mdash;Booting from DVD</title>
+   <para>
+    This procedure describes booting from a DVD as an example, but you can
+    also use another local installation medium like an ISO image on a USB mass
+    storage device. The way to select the boot method and to start up the
+    system from the medium depends on the system architecture and on whether
+    the machine has a traditional BIOS or UEFI. For details, see the links
+    below.
+   </para>
+   <step>
+    <para>
+     Insert DVD 1 of the &sle; 12 SP2 installation media and boot your
+     machine. A <guimenu>Welcome</guimenu> screen is displayed, followed by
+     the boot screen.
+    </para>
+   </step>
+   <step>
+    <para>
+     Select the type of network installation source you want to use (FTP,
+     HTTP, NFS, SMB, or SLP). Usually you get this choice by pressing
+     <keycap>F4</keycap>, but in case your machine is equipped with UEFI
```

Comment:
This is not about the BIOS/UEFI but about the IP configuration in the SLE installer.